lighthouse/blueprints/cherrypicked_plates.py

```diff
@@ -28,9 +29,16 @@
 def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
 
     try:
+        user_id = request.args.get("user_id", "UNKNOWN")
+
         barcode = request.args.get("barcode", "")
         if len(barcode) == 0:
-            return invalid_url_error()
+            return missing_barcode_url_error()
```

Comment:
I think for all these helper methods that return a single error and a `HTTPStatus.BAD_REQUEST`, you could add a single method like `bad_request_response_with_error(error)` that can be called with a different error each time.

Also could be used for when `if not mongo_samples:` and `if not plate_id_mappings:` evaluate to True

Reply:
Good point., done this